### ISSUE TITLE
Prevent admin delete other admin

### ISSUE THREAD

**Issue by hellfish2** [amnah/yii2-user#198]
Hi to all,

I was wondering if this module could be extended to prevent delete the same user role in brackend?

it's possible?

Thanks,

**Comment by amnah** [comment]
Yes, you could [extend the controller](https://github.com/amnah/yii2-user#how-do-i-extend-this-package) and then override the [actionDelete](https://github.com/amnah/yii2-user/blob/master/controllers/AdminController.php#L147-L166)
